fw/http_msg.c

```diff
@@ -1551,11 +1563,39 @@ tfw_h2_msg_cutoff_headers(TfwHttpResp *resp, TfwHttpRespCleanup* cleanup)
 
 			if ((begin <= off) && (end >= off)) {
```

Comment:
```suggestion
			if ((begin > off) || (end < off))
			        continue;
```

Reply:
I believe we can't skip the iteration here because there might be fragments in the current skb.